docs/tutorials/contributing/guidelines.rst

```diff
@@ -11,11 +11,15 @@ The project follows the following guidelines:
 2. Try as best as possible to follow `PEP8
    <https://www.python.org/dev/peps/pep-0008/>`_ which includes **using
    descriptive variable names**.
-3. Testing: the project uses the `unittest
+3. Commits: Please try to use commit messages that give a meaningful history
+   for anyone using git's log features. Try to use messages that complete sentence,
+   "If merged, this commit will...." There is some excellent guidance on the subject
```

Comment:
How about "This commit <adds feature X> <fixes issue Y> <corrects typo> <etc.>" rather than "if merged, ..." which seems redundant.